frontend/src/components/Signup.js

```diff
@@ -0,0 +1,146 @@
+import React, { useState } from 'react'
+import { useDispatch } from "react-redux"
+import { auth } from "../reducers/auth"
+import styled from 'styled-components/macro'
+
+const Wrapper = styled.section`
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  margin: 0 auto;
+  width: 100%;
+  min-height: 100vh;
+  padding: 10px;
+`;
+const SignupForm = styled.form`
+display: flex;
+flex-direction: column;
+justify-content: center;
+align-items: center;
+border-radius: 5px;
+width: 500px;
+height: 600px;
+border: 1px solid #000;
+@media (min-width: 768px) {
+  
+  }
+
+`
+const Title = styled.h1`
+color: #47476b;
+text-transform: uppercase;
+font-size:40px;
+`
+const ErrorText = styled.h2`
+color: red;
+`
+const Input = styled.input`
+border-style: hidden;
+border-radius: 5px;
+border-bottom: 2px solid black;
+background-color: #eee;
+width: 100%;
+height: 40px;
+font-size: 20px;
+line-height: 20px;
+display: flex;
+align-items: center;
+justify-content: center;
+flex-direction: column;
+padding: 0px 10px;
+@media (min-width: 768px) {
+   
+}
+`
+
+const Button = styled.button`
+background: transparent;
+color: #47476b;
+border: 3px solid #47476b;
+padding: 15px;
+border-radius: 20px;
+text-transform: uppercase;
+font-size:14px;
+margin: 15px;
+&:hover {
+  background: #eee;
+  cursor: pointer;
+}
+`
+const LabelText = styled.label`
+margin-bottom: 10px;
+margin-top: 5px;
+font-size: 18px;
```

Comment:
I feel like these css styles should be moved to their own files.  Perhaps that would allow you to reuse some of the styling on each component as well!